### PR TITLE
Remove watchdog from Dockerfile

### PR DESCRIPTION
**Pull request by SmithPeder**
* Remove watchdog from Dockerfile
* Bump node to 18 :sunglasses:
* Change backend name from kaaf to server
* Split dependencies into base and prod
* Change endpoint from /kaaf to /generate
* Wrap mail code in if(prod), and use consolemail
* Only import sentry and mail in prod

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,18 +1,14 @@
...
-RUN chmod +x /usr/bin/fwatchdog
+COPY ./server/requirements/ ./server/requirements/
 
-COPY ./kaaf/req.txt ./kaaf/req.txt
+RUN pip install --no-cache-dir -r server/requirements/prod.txt
 
-RUN pip install --no-cache-dir -r kaaf/req.txt
-
-FROM node:16-slim AS build-frontend
+FROM node:18-slim AS build-frontend
```
Comment (ivarnakken) [inline Dockerfile:11]: 😎

Comment (SmithPeder) [reply Dockerfile:11]: WE 18 NOW!!!

File: server/handler.py [hunk 2]
```diff
@@ -196,16 +196,30 @@ def handle(data):
 
     try:
         file = create_pdf(data)
-        mail.send_mail([data["mailto"], data["mailfrom"]], data, file)
+        if os.environ.get("ENVIRONMENT") == "production":
+            try:
+                import mail
```
Comment (ivarnakken) [inline server/handler.py:201]: I'm skeptical, but let's give it a try 🙃

**Review by ivarnakken — APPROVED** [review]
:shipit:

**Review by PeterJFB — APPROVED** [review]
Remove server/server.py and then we good:)

**Comment by SmithPeder** [comment]
> Remove server/server.py and then we good:)

Good catch, I had accidentally deleted it with some bad git commands 🙃

**Pull request merged**